src/WorkOS.net/Services/MFA/MFAService.cs

```diff
@@ -0,0 +1,40 @@
+namespace WorkOS
+{
+    using System.Net.Http;
+    using System.Threading.Tasks;
+
+    public class MFAService : Service
```

Comment:
Could we name this `MfaService` to match our new casing conventions for initialisms?

```suggestion
    public class MfaService : Service
```